resources/lang/pt_pt.json

```diff
@@ -1,44 +1,5 @@
 {
```

Comment:
Why removing all of these lines?

Reply:
I used just the ones on en.json.

Reply:
@joseborges OK I took a look and the `en.json` is not the correct source; these phrases, or at least many of them, are still in use. Could you bring them back in this PR, and I'll make a note to fix `en.json` later?

Reply:
the pl.json file has it all ;)